internal/common/testdata/exported_blueprint.json

```diff
@@ -1,8 +1,8 @@
 {
   "content_sources": [
     {
-      "gpg_key": "some-gpg-key",
-      "name": "payload",
```

Comment:
(nitpick) here the name changes from "payload" -> "baseos" which is good but the commit message is a bit terse and just says " v1/tests: don't declare constants which are only used once which is not the full picture of this commit AFAICT